src/server/worldserver/worldserver.conf.dist

```diff
@@ -3816,9 +3816,9 @@ Logger.mmaps=3,Server
 #Logger.ahbot=3,Console Server
 #Logger.auctionHouse=3,Console Server
 #Logger.bg.arena=3,Console Server
-#Logger.bg.battlefield=3,Console Server
 #Logger.bg.battleground=3,Console Server
 #Logger.bg.reportpvpafk=3,Console Server
+#Logger.battlefield=3,Console Server
```

Comment:
This is mindly infuriating, please order it alphabetically before my mind explodes >.<